pallets/multicollateral-bonding-curve-pool/src/tests.rs

```diff
@@ -3354,12 +3396,12 @@ mod tests {
             MockDEXApi::add_reserves(vec![(VAL, balance!(0.4)), (TBCD, balance!(1000))]).unwrap();
 
             // val is not enough for one of distributions, it's still present
-            MBCPool::on_initialize(RETRY_DISTRIBUTION_FREQUENCY.into());
+            run_to_block(RETRY_DISTRIBUTION_FREQUENCY as u64 * 3);
             let free_reserves_balance_4 =
                 Assets::free_balance(&USDT, &MBCPool::free_reserves_account_id().unwrap()).unwrap();
             assert_eq!(
-                MBCPool::pending_free_reserves(),
-                vec![(USDT, second_pending_balance)]
+                MBCPool::pending_free_reserves(RETRY_DISTRIBUTION_FREQUENCY as u64 * 3 + 2),
```

Comment:
I think it is better to change type of `RETRY_DISTRIBUTION_FREQUENCY` to `u64`.

Reply:
Added constant to Config trait